charts/linkerd2/README.md

```diff
@@ -0,0 +1,164 @@
+# Linkerd2 Helm Chart
+
+Linkerd is a *service mesh*, designed to give platform-wide observability,
+reliability, and security without requiring configuration or code changes.
+
+Linkerd is a Cloud Native Computing Foundation ([CNCF][cncf]) project.
+
+## Quickstart and documentation
+
+You can run Linkerd on any Kubernetes 1.12+ cluster in a matter of seconds. See
+the [Linkerd Getting Started Guide][getting-started] for how.
+
+For more comprehensive documentation, start with the [Linkerd
+docs][linkerd-docs].
+
+## Prerequisite: identity certificates
+
+The identity component of Linkerd requires setting up a trust anchor
+certificate, and an issuer certificate with its key. These need to be provided
+to Helm by the user (unlike when using the `linkerd install` CLI which can
+generate these automatically). You can provide your own, or follow [these
+instructions](https://linkerd.io/2/tasks/generate-certificates/) to generate new ones.
+
+Note that the provided certificates must be ECDSA certficates.
+
+## Adding Linkerd's Helm repository
+
+```bash
+# To add the repo for Linkerd2 stable releases:
+helm repo add linkerd-stable https://run.linkerd.io/stable
```

Comment:
Can we call the repo `linkerd`, instead of `linkerd-stable`?

Reply:
so one repo would be called `linkerd` and the other `linkerd-edge`?

Reply:
Yeah, or `linkerd.io`? It's just that other repos I have seen don't add the `-stable` suffix. (I do see others that have a `-dev` suffix that serve the same purpose as our edge repo.).